pkg/auth/dependency/webapp/sso_state.go

```diff
@@ -0,0 +1,14 @@
+package webapp
+
+type SSOState map[string]string
```

Comment:
Right now we only need to preserve Auth UI's `redirect_uri`, but we are going to have more query parameters we want to preserve. For example, [This PR](https://github.com/SkygearIO/skygear-server/pull/1326) adds support for `ui_locales`. It is more future-proof to persist the entire encoded query. (`url.Values.Encode()`)